Logging/Log.cs

```diff
@@ -104,15 +106,15 @@ public static void Error(Exception exception, string message = null, bool overri
         /// <summary>
         /// Log trace
         /// </summary>
-        public static void Trace(string traceText, bool overrideMessageFloodProtection = false) 
-        { 
-            try 
+        public static void Trace(string traceText, bool overrideMessageFloodProtection = false)
```

Comment:
I think we need the no inlining attribute here and on debug as well.

Reply:
Sorry about these changes, they were automatic, just deleting spaces.
The only two places that `WhoCalledMe.GetMethodName()` are used, is in `Error()` and in the test